tutorial/10_key_features/003_efficient_optimization_algorithms.py

```diff
@@ -80,6 +80,8 @@
 #
 # - Threshold pruning algorithm implemented in :class:`~optuna.pruners.ThresholdPruner`
 #
+# - A pruning algorithm based on `Wilcoxon signed-rank test <https://en.wikipedia.org/wiki/Wilcoxon_signed-rank_test>`_ implemented in :class:`~optuna.pruners.WilcoxonPruner`
```

Comment:
This link is also not rendered as intended.
<img width="727" alt="image" src="https://github.com/optuna/optuna/assets/30489874/5be9cdbd-635b-48e1-8096-d9f51259911d">